template-dockerfile.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
I like the approach, but my only worry is, will this tooling work on a windows machine? we are using shell scripts with Linux commands to generate dockerfiles

Reply:
The current build tooling [Makefile](https://github.com/vmware-tanzu/build-tooling-for-integrations/blob/main/templates/Makefile) fails to run in a windows environment. Is the expectation to enable build-tooling cross-platform?

Reply:
If the build tooling in its current state is not compatible with the windows environment, I think, making it compatible should be tracked as a different GH issue.

Reply:
+1

Reply:
we have an issue created for that https://github.com/vmware-tanzu/build-tooling-for-integrations/issues/51

Reply:
Thanks @yharish991! So, should we track the task of making the newly added script windows compatible in https://github.com/vmware-tanzu/build-tooling-for-integrations/issues/51

Reply:
Resolving as these changes are not required now